teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md

```diff
@@ -0,0 +1,189 @@
+---
+Module Name: microsoftteams
+applicable: Microsoft Teams
+title: Grant-CsTeamsMediaConnectivityPolicy
+online version: https://learn.microsoft.com/powershell/module/teams/Grant-CsTeamsMediaConnectivityPolicy
+schema: 2.0.0
+author: lirunping_MSFT
+ms.author: runli
+---
+
+# Grant-CsTeamsMediaConnectivityPolicy
+
+## SYNOPSIS
+This cmdlet applies an instance of the Teams media connectivity policy to users or groups in a tenant.
+
+## SYNTAX
+
+### Identity (Default)
+```
+Grant-CsTeamsMediaConnectivityPolicy [<CommonParameters>]
+```
+
+### GrantToUser
+```
+Grant-CsTeamsMediaConnectivityPolicy -Identity <String> [[-PolicyName] <String>] [<CommonParameters>]
+```
+
+### GrantToGroup
+```
+Grant-CsTeamsMediaConnectivityPolicy [[-PolicyName] <String>] [-Group] <String> -Rank <Int32>
+ [<CommonParameters>]
+```
+
+### GrantToTenant
+```
+Grant-CsTeamsMediaConnectivityPolicy [[-PolicyName] <String>] [-Global] [-Force] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet applies an instance of the Teams media connectivity policy to users or groups in a tenant.
+
+Passes in the `Identity` of the policy instance in the `PolicyName` parameter and the user identifier in the `Identity` parameter or the group name in the `Group` parameter. One of either `Identity` or `Group` needs to be passed.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -PolicyName Test -Identity testuser@test.onmicrosoft.com
+```
+
+Assigns a given policy to a user.
+
+### Example 2
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Group f13d6c9d-ce76-422c-af78-b6018b4d9c80 -PolicyName Test
+```
+
+Assigns a given policy to a group.
+
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName Test
+```
+
+Assigns a given policy to the tenant.
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName Test
```

Comment:
```suggestion
PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName $null
```

Reply:
The "Test" in the example is an identify for the policy name. May I ask why it has to be $null ? Does it mean I need to change all the "Test" to $null in the examples of this file ?

Reply:
I'll keep this as-is. @shivtorov please let me know if you prefer this to be changed throughout the file.